### PR TITLE
[action] Add bot GH signing key for auto docs update

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -180,13 +180,28 @@ jobs:
           repository: smallstep/docs
           token: ${{ secrets.DOCS_PAT }}
           path: './docs'
+      - name: Setup bot SSH signing key
+        uses: webfactory/ssh-agent@v0.5.3
+        env:
+          has_SSH_PRIVATE_KEY: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
+        if: ${{ env.has_SSH_PRIVATE_KEY != '' }}
```
Comment (maraino) [inline .github/workflows/release.yml:187]: Should it be 
```suggestion
        env:
          HAS_SSH_PRIVATE_KEY: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
        if: ${{ env.HAS_SSH_PRIVATE_KEY != '' }}
```

Comment (maraino) [reply .github/workflows/release.yml:187]: Or can you do the if with the secret?
```suggestion
        if: ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY != '' }}
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -180,13 +180,28 @@ jobs:
...
+        with:
+          ssh-private-key: |
+            ${{ secrets.STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY }}
+            STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY
```
Comment (maraino) [inline .github/workflows/release.yml:191]: Do we need this extra line (`STEP_TRAVIS_CI_GH_PRIVATE_SIGNING_KEY`)?

Comment (maraino) [reply .github/workflows/release.yml:191]: Should it be a second SSH key from a secret?

**Review by maraino — COMMENTED** [review]
I've added a suggestion, and there's one line unclear about what it does.

File: .github/workflows/release.yml (hunk 1 above)
Comment (dopey) [reply .github/workflows/release.yml:187]: Unfortunately you can't use secrets in conditionals. But the code is wrong. I should be comparing to `true`.

**Review by maraino — APPROVED** [review]

**Pull request merged**